libs/ojsonnet/Eval_jsonnet.ml

```diff
@@ -190,7 +190,16 @@ and eval_expr (env : V.env) (v : expr) : V.value_ =
                    fst field.fld_name = fld)
           with
           | None -> error tk (spf "field '%s' not present in %s" fld (sv e))
-          | Some fld -> evaluate_lazy_value_ fld.fld_value)
+          | Some fld -> (
+              let new_self = V.Object (_l, (_assertsTODO, fields), _r) in
```

Comment:
Why we don't simply reuse 'e' instead of building this new_self? or just do
let new_self = e (or use 'as o' in the pattern to bind the value in the V.Object ...)